### PR TITLE
Form create user

### PR DESCRIPTION
**Pull request by LeoDiduch**

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -26,6 +27,7 @@ function App() {
         <Route path="/ambassadors" component={Ambassadors}></Route>
         <Route path="/causes/:id" component={CauseInfos}></Route>
         <Route path="/causes" component={Causes}></Route>
+        <Route path="/admin" component={FormAdmin}></Route>
```
Comment (TheoBoschetti) [inline frontend/src/App.js]: I think it will be better to use private Route here

File: frontend/src/components/FormCreateUser.jsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+      password: "",
+      email: "",
+      canAdmin: false,
+      canCreateContent: false
+    });
+  };
+
+  const handleSubmit = () => {
+    const data = user;
+    console.log(apiCall);
```
Comment (TheoBoschetti) [inline frontend/src/components/FormCreateUser.jsx]: no console log

File: frontend/src/components/FormCreateUser.jsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    axios
+      .post(`${apiCall}/auth`, data)
+      .then(res => {
+        console.log(res.data);
```
Comment (TheoBoschetti) [inline frontend/src/components/FormCreateUser.jsx]: no console log

File: frontend/src/components/FormCreateUser.jsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+        console.log("Booyah!");
```
Comment (TheoBoschetti) [inline frontend/src/components/FormCreateUser.jsx]: Same

File: frontend/src/components/FormCreateUser.jsx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+        resetForm();
+      })
+      .catch(err => {
+        console.log(err);
```
Comment (TheoBoschetti) [inline frontend/src/components/FormCreateUser.jsx]: An should be better

**Review by TheoBoschetti — CHANGES_REQUESTED** [review]

File: frontend/src/components/FormCreateUser.jsx (hunk 2 above)
Comment (TheoBoschetti) [inline frontend/src/components/FormCreateUser.jsx]: non

**Review by TheoBoschetti — DISMISSED** [review]
ok

File: frontend/src/components/FormCreateUser.jsx [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+            onChange={event => {
+              setUser({
+                ...user,
+                canCreateContent: !user.canCreateContent
+              });
+            }}
+          />
+        </div>
+
+        <input type="submit" onClick={handleSubmit} />
```
Comment (LoicBrassart) [inline frontend/src/components/FormCreateUser.jsx]: No need to create an explicit `onClick` event: it'd be better to add an `onSubmit` on the `form` tag: it'd be more readable, and would work when the user types the "Return" key

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
A trigger to move, but no Evil spotted beyond that

File: frontend/src/components/FormCreateUser.jsx (hunk 6 above)
Comment (LeoDiduch) [reply frontend/src/components/FormCreateUser.jsx]: OK

**Review by BastienSaulnier — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**